app/controllers/application_controller.rb

```diff
@@ -71,11 +71,11 @@ def search_invoked?
 
   def set_search
     # rubocop:disable Naming/MemoizedInstanceVariableName
-    @search ||= Search.new(search_attributes)
+    @search ||= Search.new(search_params)
```

Comment:
just checking, do we need to set this an instance var?
I noticed that we are using `f` in the view (but I may miss other uses)